### PR TITLE
Increase contrast of Checkbox, Radio, And Switch widgets

### LINKED ISSUE #16394: [a11y] Checkboxes should be blue 600
For sufficient color contrast checkboxes (and similar items) should be blue600. They currently blue500.

### PR DESCRIPTION
**Pull request by jonahwilliams**
- Adds toggleableActiveColor to ThemeData which uses the 600 shade instead of 500.
- Updates Checkbox, Radio, and Switch to use this instead of accent color.

It looks like the contrast on sliders is okay currently.

Fixes: https://github.com/flutter/flutter/issues/16394

## Switch

| Old        | New           |
| ------------- |-------------| 
|   |   | 

## Radio

| Old        | New           |
| ------------- |-------------| 
|   |   | 

## Checkbox

| Old        | New           |
| ------------- |-------------| 
|   |   |

### CODE REVIEW

File: packages/flutter/lib/src/material/checkbox.dart [hunk 1]
```diff
@@ -98,7 +98,7 @@ class Checkbox extends StatefulWidget {
 
   /// The color to use when this checkbox is checked.
   ///
-  /// Defaults to accent color of the current [Theme].
+  /// Defaults to [Theme.toggleableActiveColor].
```
Comment (HansMuller) [inline packages/flutter/lib/src/material/checkbox.dart]: ThemeData

File: packages/flutter/lib/src/material/radio.dart [hunk 2]
```diff
@@ -93,7 +93,7 @@ class Radio<T> extends StatefulWidget {
 
   /// The color to use when this radio button is selected.
   ///
-  /// Defaults to accent color of the current [Theme].
+  /// Defaults to [Theme.toggleableActiveColor].
```
Comment (HansMuller) [inline packages/flutter/lib/src/material/radio.dart]: ThemeData

I'll stop repeating this feedback. There are more to fix...

File: packages/flutter/lib/src/material/theme_data.dart [hunk 3]
```diff
@@ -402,6 +407,10 @@ class ThemeData extends Diagnosticable {
   /// The default fill color of the [Material] used in [RaisedButton]s.
   final Color buttonColor;
 
+  /// The color used to highlight the active states of toggleable widgets like
+  /// [Switch], [RadioButton], and [Checkbox].
```
Comment (HansMuller) [inline packages/flutter/lib/src/material/theme_data.dart]: [Radio]

**Review by HansMuller — APPROVED** [review]
LGTM with a few typos to fix

File: packages/flutter/lib/src/material/checkbox.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/flutter/lib/src/material/checkbox.dart]: Woops, fixed

File: packages/flutter/lib/src/material/radio.dart (hunk 2 above)
Comment (jonahwilliams) [reply packages/flutter/lib/src/material/radio.dart]: Fixed

File: packages/flutter/lib/src/material/theme_data.dart (hunk 3 above)
Comment (jonahwilliams) [reply packages/flutter/lib/src/material/theme_data.dart]: Fixes

Comment (jonahwilliams) [reply packages/flutter/lib/src/material/theme_data.dart]: I mean Fixed!

**Pull request merged**